app/views/jobs/getjob.html.erb

```diff
@@ -1,55 +1,167 @@
 <% content_for :javascripts do %>
   <%= javascript_include_tag "sorttable" %>
 <% end %>
-
-<h1>Job <%= @job[:name]%></h1>
 <p>
 <%= link_to "Return to Jobs", {:action=>"index"} %>
 </p>
-<h2>Summary</h2>
-<ul>
-<li>Job ID: <%= @job[:id] %>
-<li>State: <%= @job[:state] %>
-<li>Status: <%= @job[:status] %>
-<li>Elapsed time: <%= @job[:elapsed] %> secs
-<li>VM instance: <%= @job[:rjob]["vm"]["id"] %>
-</ul>
+<h3>Job <%= @job[:id] %> Summary</h3>
+<div class="jobs-table-div">
+  <table class="jobs-table">
+    <tbody>
+      <tr>
+        <td class="left">
```

Comment:
To avoid repeatedly writing the classes, perhaps we could do

```css
.jobs-table td:first-child {
  width: 30%;
}
.jobs-table td:last-child {
  width: 70%;
}
```

Also not sure if you're aware, but `left` and `right` are existing classes that add floats.

By removing the `left` and `right` classes, there is the added effect of vertically centering the `notifyURL` text, which might be desirable

Reply:
did not know about `left` and `right` adding floats, thanks for letting me know. Switched it to your suggestion and moved the style code into a new stylesheet so that I stop adding bloat to `styles.scss.css`